### PR TITLE
Add test related to launcher capabilities

### PR DESCRIPTION
**Pull request by kbidarkar**
Signed-off-by: Kedar Bidarkar <kbidarka@redhat.com>

**What this PR does / why we need it**: `Add test related to launcher capabilities`

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**: `Add test related to launcher capabilities`

**Release note**:

```
NONE
```

### CODE REVIEW

File: tests/security_features_test.go [hunk 1]
```diff
@@ -188,4 +188,39 @@ var _ = Describe("SecurityFeatures", func() {
...
+				if containerSpec.Name == "compute" {
+					container = containerSpec
+					break
+				}
+			}
+			caps := *container.SecurityContext.Capabilities
+			Expect(len(caps.Add)).To(Equal(3))
+
+			By("Checking virt-launcher Pod's compute container has precisely the documented extra capabilities")
+			for _, cap := range caps.Add {
```
Comment (xpivarc) [inline tests/security_features_test.go:220]: You can use ContainElements matcher for this.

Comment (kbidarkar) [reply tests/security_features_test.go:220]: Thanks for the review comments, I am new to golang/gomega.

I looked at making use of this and I realized we need gomega version `v1.9.0` and I think we are using `v1.7.0`. Maybe we need to update gomega to make use of `ContainElements`.

Comment (xpivarc) [reply tests/security_features_test.go:220]: I see... I don't have a strong opinion about this. Yes, we are using gomega `1.7.0`. I think we can update it.

File: tests/security_features_test.go [hunk 2]
```diff
@@ -188,4 +188,39 @@ var _ = Describe("SecurityFeatures", func() {
 			})
 		})
 	})
+
+	Context("Check virt-launcher capabilities", func() {
+		var container k8sv1.Container
+		var vmi *v1.VirtualMachineInstance
+
+		It("has precisely the documented extra capabilities relative to a regular user pod", func() {
```
Comment (ILpinto) [inline tests/security_features_test.go:196]: Missing test id

Comment (kbidarkar) [reply tests/security_features_test.go:196]: Ah! my bad, will add

Comment (kbidarkar) [reply tests/security_features_test.go]: Added the relevant test_id.

**Pull request merged**